azure-pipelines-azurezagent/files/start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+export AGENT_ALLOW_RUNASROOT="1"
+
+print_header() {
+  lightcyan='\033[1;36m'
+  nocolor='\033[0m'
+  echo -e "${lightcyan}$1${nocolor}"
+}
+
+source ./env.sh
+
+print_header "Running Azure Pipelines agent..."
+
+trap 'exit 0' EXIT
+trap 'exit 130' INT
+trap 'exit 143' TERM
+
+chmod +x ./run.sh
+
+# To be aware of TERM and INT signals call run.sh
+# Running it with the --once flag at the end will shut down the agent after the build is executed
+./run.sh "$@" & wait $!
```

Comment:
I see it also has `run-docker.sh` file? Should we use that?

Reply:
Why?

Reply:
The file is also is different and as it explicitly mentions docker? Not sure if the difference is relevant?